app/models/concerns/attribute_displayable.rb

```diff
@@ -0,0 +1,13 @@
+module AttributeDisplayable
```

Comment:
Actually, Having some guidance for what goes in any given concern vs in the main patient might be good for all of them.